UniversalFramework_Base.xcconfig

```diff
@@ -11,7 +11,7 @@ VALID_ARCHS[sdk=macosx*]               = i386 x86_64
 VALID_ARCHS[sdk=iphoneos*]             = arm64 armv7 armv7s
 VALID_ARCHS[sdk=iphonesimulator*]      = arm64 armv7 armv7s
 VALID_ARCHS[sdk=watchos*]              = armv7k
-VALID_ARCHS[sdk=watchsimulator*]       = armv7k
+VALID_ARCHS[sdk=watchsimulator*]       = arm64 armv7
```

Comment:
Why is the change for the watch simulator architecture needed? I don't think Xcode even cares about those values, because the watch simulator binaries should be built for `i386`.